docs/admin/node-allocatable.md

```diff
@@ -9,115 +9,182 @@ title: Reserving Compute Resources for System Daemons
-[This blog post](http://blog.kubernetes.io/2016/11/visualize-kubelet-performance-with-node-dashboard.html) explains how the dashboard can be interpreted to come up with a suitable `kube-reserved` reservation.
-
-To optionally enforce `kube-reserved` on system daemons, specify the parent control group for kube daemons as the value for `--kube-reserved-cgroup` kubelet flag.
-
-It is recommended that the kubernetes system daemons are placed under a top level control group (`runtime.slice` on systemd machines for example).
-Each system daemon should ideally run within its own child control group.
-Refer to [this doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node-allocatable.md#recommended-cgroups-setup) for more details on recommended control group hierarchy.
-
-Note that Kubelet **does not** create `--kube-reserved-cgroup` if it doesn't exist. Kubelet will fail if an invalid cgroup is specified.
+`kube-reserved` is typically a function of `pod density` on the nodes. [This
+performance dashboard](http://node-perf-dash.k8s.io/#/builds) exposes `cpu` and
+`memory` usage profiles of `kubelet` and `docker engine` at multiple levels of
+pod density. [This blog
+post](http://blog.kubernetes.io/2016/11/visualize-kubelet-performance-with-node-dashboard.html)
+explains how the dashboard can be interpreted to come up with a suitable
+`kube-reserved` reservation.
+
+To optionally enforce `kube-reserved` on system daemons, specify the parent
+control group for kube daemons as the value for `--kube-reserved-cgroup` kubelet
+flag.
+
+It is recommended that the kubernetes system daemons are placed under a top
+level control group (`runtime.slice` on systemd machines for example). Each
+system daemon should ideally run within its own child control group. Refer to
+[this
+doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node-allocatable.md#recommended-cgroups-setup)
+for more details on recommended control group hierarchy.
+
+Note that Kubelet **does not** create `--kube-reserved-cgroup` if it doesn't
+exist. Kubelet will fail if an invalid cgroup is specified.
 
 ### System Reserved
 
 - **Kubelet Flag**: `--system-reserved=[cpu=100mi][,][memory=100Mi]`
 - **Kubelet Flag**: `--system-reserved-cgroup=`
 
 
-`system-reserved` is meant to capture resource reservation for OS system daemons like `sshd`, `udev`, etc.
-`system-reserved` should reserve `memory` for the `kernel` too since `kernel` memory is not accounted to pods (yet) in Kubernetes.
-Reserving resources for user login sessions is also recommended (`user.slice` in systemd world).
+`system-reserved` is meant to capture resource reservation for OS system daemons
+like `sshd`, `udev`, etc. `system-reserved` should reserve `memory` for the
+`kernel` too since `kernel` memory is not accounted to pods (yet) in Kubernetes.
```

Comment:
What do you mean by 'yet' here? Is this a future feature or something that will happen later in this process? Maybe clarify or remove to avoid confusion